### ISSUE TITLE
Failed to deploy path that does not exist

### ISSUE THREAD

**Issue by Ausxor** [microsoft/vscode-azurefunctions#2904]
VsCode: 1.59.0 (User Setup)
Azure Functions Extension: 1.4.1

**Steps to Replicate:**
1. On Windows, open Visual Studio Code
1. Ctrl-Shift-P (Command Palette)
1. Select ‘Azure Functions Create New Project’ -> Select Any Folder -> C# -> .NET Core 3 -> HTTP Trigger -> Default Trigger Name -> Default Namespace -> ’Function’ Access Rights
1. Simulate a failure to connect to nuget.org. I had this issue because on a fresh install, ‘nuget.org’ was not setup as a default package source for some reason. Either:
    * Add the following to your hosts file
     ```
    127.0.0.1       nuget.org
    127.0.0.1       api.nuget.org
    ```
    * Remove nuget.org as a package source:
    ```
    cmd > dotnet nuget remove source -n nuget.org
    ```
1. Delete these folders if they exist:
    * C:\Users\\\<Username\>\\.nuget\packages\microsoft.net.sdk.functions
    * <project_root>\bin
1. Navigate to the Azure tab of vscode, and deploy the app. Select ‘+ Create New Function App in Azure’ -> .NET Core 3.1 -> Australia East (Shouldn't matter)
1. You will observe an error. Press ‘Cancel’.
    ![Picture1](https://user-images.githubusercontent.com/4426330/128464531-e59e1acf-6875-4ce7-8ad1-66dd6f46fa87.png)

1. Reverse the changes to block nuget package restore. Make sure nuget.org is available and listed as a package source.
1. Repeat the deployment. This time select the already created Function App.

**Expected:**
Deployment will complete successfully

**Actual**
Deployment fails with the following error.
![Screenshot 2021-08-06 161438](https://user-images.githubusercontent.com/4426330/128464789-e3e86fa1-7ea3-48be-bbc8-ce877ee288a3.png)

**Related**
https://github.com/microsoft/vscode-azurefunctions/issues/907
https://github.com/microsoft/vscode-azurefunctions/issues/1419

**Comment by ejizba** [comment]
I did some investigations and I believe this is a regression related to the release of VS Code 1.59. One workaround is to downgrade to v1.58

I've filed the following issue on VS Code: https://github.com/microsoft/vscode/issues/130277

**Comment by ejizba** [comment]
This has been fixed for me on VS Code 1.59.1
